### PR TITLE
Update explicit.md to conform language specification

### LINKED ISSUE #6952: Disagreement with ECMA-334
@flaviotsales commented on [Fri Aug 10 2018](https://github.com/dotnet/docs.pt-br/issues/62)

The two comments
~~~
// Must be defined inside a class called Fahrenheit:
~~~
disagrees with the ECMA-334 standard on 11.5.2 Permitted user-defined conversions, which says

&gt;* Either S0 or T0 is the class or struct type in which the operator declaration takes place.

---
#### Detalhes do documento

⚠ *Não edite esta seção. É necessário para a vinculação do problema do docs.microsoft.com ➟ GitHub.*

* ID: 265f75ba-8fe1-3586-6a53-e9a50acadab0
* Version Independent ID: 2f8d9e81-3004-8cef-2161-79b800a6538c
* Content: [Palavra-chave explicit (Referência de C#)](https://docs.microsoft.com/pt-br/dotnet/csharp/language-reference/keywords/explicit)
* Content Source: [docs/csharp/language-reference/keywords/explicit.md](https://github.com/dotnet/docs.pt-br/blob/live/docs/csharp/language-reference/keywords/explicit.md)
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by pkulikov**
Fixes #6952

Moved comment into the article text, so it will be localized.
Snippet is updated by dotnet/samples#254

### CODE REVIEW

File: docs/csharp/language-reference/keywords/explicit.md [hunk 1]
```diff
@@ -10,11 +10,13 @@ ms.assetid: cfb8f42a-e411-4db2-af9b-796b05644846
 ---
 # explicit (C# Reference)
 
-The `explicit` keyword declares a user-defined type conversion operator that must be invoked with a cast. For example, this operator converts from a class called Fahrenheit to a class called Celsius:
+The `explicit` keyword declares a user-defined type conversion operator that must be invoked with a cast.
+
+The following example defines the operator that converts from a `Fahrenheit` class to a `Celsius` class (the operator must be defined either inside a `Fahrenheit` class or a `Celsius` class):
```
Comment (rpetrusha) [inline docs/csharp/language-reference/keywords/explicit.md]: It would be best to make "(the operator must be defined..." a separate sentence.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build after modified example merged.

File: docs/csharp/language-reference/keywords/explicit.md [hunk 2]
```diff
@@ -48,4 +50,4 @@ The following example defines a struct, `Digit`, that represents a single decima
 [implicit](implicit.md)  
 [operator (C# Reference)](operator.md)  
 [How to: Implement User-Defined Conversions Between Structs](../../programming-guide/statements-expressions-operators/how-to-implement-user-defined-conversions-between-structs.md)
```
Comment (rpetrusha) [inline docs/csharp/language-reference/keywords/explicit.md]: nit unrelated to your PR: the See also items could be made bulleted list items.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @pkulikov. This is an important change, since comments aren't localized, I've suggested two changes, one unrelated to your PR.

**Comment by pkulikov** [comment]
@rpetrusha thank you for review; I've addressed your feedback.

**Review by rpetrusha — APPROVED** [review]
Thanks for making the additional changes, @pkulikov. I'll merge your PR now.

**Pull request merged**